### DISCUSSION TITLE
feat: io_uring implementation

### DISCUSSION THREAD

**Discussion by ryanrussell** [seaweedfs/seaweedfs#3697] [Ideas]
@chrislusf 

Curious if `io_uring` is in the roadmap for `seaweedfs`?

I'm not familiar with the `golang` implementation, but I did see one at https://github.com/Iceber/iouring-go

Is this something you see in the future for seaweed?

Best,
Ryan

**Comment by chrislusf** [comment]
There was an attempt before https://github.com/seaweedfs/seaweedfs/pull/2374

**Reply by ryanrussell** [reply]
Got it. Sounds like it wasn't a meaningful win in the last attempt. 

Frankly, I do not have the `golang` expertise to attempt to improve on this implementation.

Just going to mark this as answered. If someone else comes along who thinks they can make a marked improvement on this last attempt, feel free to bring this back to life.

Considering closed.
